server/routes/report.ts

```diff
@@ -81,18 +81,17 @@ export default function (router: IRouter) {
           );
 
           return response.ok({
-            body: stream,
-            headers: {
-              'content-type': 'image/png',
-              'content-disposition': `attachment; filename=${fileName}.${reportParams.report_format}`,
+            body: {
+              data: stream,
+              filename: `${fileName}.${reportParams.report_format}`,
```

Comment:
from your changes in `reportHelpers`, the `filename` here already includes the suffix.(.pdf / .png)

Reply:
getFileName only includes suffix for pdf so png is left unchanged